src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -96,7 +97,8 @@ class RtpReceiverImpl @JvmOverloads constructor(
     private val tccGenerator = TccGeneratorNode(rtcpSender, streamInformationStore, logger)
     private val remoteBandwidthEstimator = RemoteBandwidthEstimator(streamInformationStore, logger, diagnosticContext)
     private val audioLevelReader = AudioLevelReader(streamInformationStore)
-    private val silenceDiscarder = SilenceDiscarder()
+    private val silenceDiscarder = DiscardableDiscarder("Silence discarder")
+    private val paddingOnlyDiscarder = DiscardableDiscarder("Padding-only discarder")
```

Comment:
Since these are generic discarders, we don't really know why they are discarding. E.g. if we get padding-only packets with an audio PT, they will be discarded by the silence discarder. 
Can you think of a way to make it less likely for future-us to get confused about the meaning of the stats from the different discarders?

Reply:
Well, we have stats from `PaddingTermination`, and I could add them to `AudioLevelReader` too.

More elaborately, I could change `shouldDiscard` to an enum with a reason?